imap_processing/cdf/config/imap_glows_l1a_variable_attrs.yaml

```
@@ -0,0 +1,381 @@
+  DEPEND_1: per_second
+  DEPEND_2: direct_event
+  VALIDMIN: 0
+  VALIDMAX: 300
+  CATDESC: Direct events recorded approximately per second
+  FIELDNAM: List of direct events, binned per second
+  # FORMAT is different from glows_cdf_attrs.py
+  # Changed this in order to avoid warnings
+  # Int with 10 digits
+  FORMAT: I10
+  VAR_TYPE: data
+  DISPLAY_TYPE: time_series
+  LABLAXIS: Counts
+
+histogram_attrs:
+  <<:  *default_attrs
+  VALIDMIN: 0
+  VALIDMAX: 70
+  CATDESC: Histogram of photon counts
+  DEPEND_0: epoch
+  DEPEND_1: bins
+  FIELDNAM: Histogram of photon counts
+  # FORMAT is different from glows_cdf_attrs.py
+  # Changed this in order to avoid warnings
+  # Float with 4 digits
+  FORMAT: F0.5
+  DISPLAY_TYPE: time_series
+  LABLAXIS: Counts
+  FILL_VAL: *int_fillval
+  UNITS: counts
+  VAR_TYPE: data
+
+metadata_attrs: &metadata_attrs
+  <<:  *default_attrs
+  DEPEND_0: epoch
+  VALIDMIN: 0
+  VALIDMAX: 1000000000
+  DISPLAY_TYPE: time_series
+  FILL_VAL: *int_fillval
+  VAR_TYPE: metadata
+  FORMAT: I10
+  LABLAXIS: Metadata
+  RESOLUTION: ' '
+
+# not-in--dicts in generate_de_dataset
+# Should I be defining the metadata_attrs here?
+ground_software_version:
+  <<: *metadata_attrs
+  CATDESC: Ground software version in the format vXXX
+  FIELDNAM: Ground software version
+
+pkts_file_name:
+  <<: *metadata_attrs
+  CATDESC: Input filename for packets data
+  FIELDNAM: Input filename
+
+last_spin_id:
+  <<: *metadata_attrs
+  CATDESC: The ID of the last spin in the block
+  FIELDNAM: Last spin ID
+
+imap_start_time:
+  <<: *metadata_attrs
+  CATDESC: Start time of the block, in seconds, in IMAP spacecraft clock time
+  FIELDNAM: Start time in seconds in IMAP time
+  # FORMAT is different from glows_cdf_attrs.py
+  # Changed this in order to avoid warnings
+  # Can range from F0 to F0.9
+  FORMAT: F0
+
+imap_time_offset:
+  <<: *metadata_attrs
+  CATDESC: End time of the block, in seconds, in IMAP spacecraft clock time
+  FIELDNAM: End time in seconds in IMAP time
+  # FORMAT is different from glows_cdf_attrs.py
+  # Changed this in order to avoid warnings
+  # Can range from F0 to F0.9
+  FORMAT: F0.9
+
+glows_start_time:
+  <<: *metadata_attrs
+  CATDESC: Start time of the block, in seconds, in GLOWS clock time
+  FIELDNAM: Start time in seconds in GLOWS time
+  # FORMAT is different from glows_cdf_attrs.py
+  # Changed this in order to avoid warnings
+  # Can range from F0 to F0.9
+  FORMAT: F0.5
+
+glows_time_offset:
+  <<: *metadata_attrs
+  CATDESC: End time of the block, in seconds, in GLOWS clock time
+  FIELDNAM: End time in seconds in GLOWS time
+  # FORMAT is different from glows_cdf_attrs.py
+  # Changed this in order to avoid warnings
+  # Can range from F0 to F0.9
+  FORMAT: F0.5
+
+flags_set_onboard:
+  <<: *metadata_attrs
+  CATDESC: Flags for bad data [missing PPS, missing time, missing spin phase, missing spin periods, overexposure, nonmonotonic event, data collected at night, HV test, pulse test, memory error]
```

CATDESC should have a max of 80 characters (I know this was copied from my code)

```suggestion
  CATDESC: Flags for bad data 
```